### PR TITLE
Update deprecated constants.

### PR DESCRIPTION
**Pull request by pq**
/cc @nilsdoehring

### CODE REVIEW

**Comment by pq** [comment]
@kevmoo Thanks! 👍 

Updated.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+### 0.6.0+1
+
+* Updated use of deprected Dart core constants.
+
+### 0.6.0
```
Comment (kevmoo) [inline CHANGELOG.md]: @pq – looks like 0.6.0 was never shipped - https://pub.dartlang.org/packages/dart_coveralls

Sorry! Can go back to 0.6.0 – prepare for release.

**Comment by pq** [comment]
Thanks! Fixed up but I can't merge.

**Comment by nilsdoehring** [comment]
@pq Thanks! Sorry for taking so long – vacation without WiFi! Will publish to pub later today.

**Comment by pq** [comment]
No worries @nilsdoehring.  Thanks for following up.  Hope you enjoyed (or are enjoying) vacation! 🌴

**Pull request merged**